addons/point_of_sale/static/src/app/navbar/back_button/back_button.js

```diff
@@ -1,42 +0,0 @@
-import { Component, useState } from "@odoo/owl";
-import { useService } from "@web/core/utils/hooks";
-import { usePos } from "@point_of_sale/app/store/pos_hook";
-import { TicketScreen } from "@point_of_sale/app/screens/ticket_screen/ticket_screen";
-import { PaymentScreen } from "@point_of_sale/app/screens/payment_screen/payment_screen";
-
-export class BackButton extends Component {
-    static template = "point_of_sale.BackButton";
-    static props = {};
-
-    setup() {
-        this.pos = usePos();
-        this.ui = useState(useService("ui"));
-    }
-    _mobile_back_button() {
-        if (this.pos.showResultMobile && this.ui.isSmall) {
-            this.pos.showResultMobile = false;
-        } else if (this.pos.selectedCategory && this.ui.isSmall) {
-            this.pos.setSelectedCategory(this.pos.selectedCategory.parent_id?.id);
-        } else {
-            return false;
-        }
-        return true;
-    }
```

Comment:
This feature has been removed, is it wanted ? If yes, can you remove all occurrence of `showResultMobile`

Reply:
right, i removed them now